app/views/errors/forbidden.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class='main'>
+  <h1>Woops!</h1>
+  <p>You do not have access to this page.</p>
+  <p>  Are you <%= link_to 'signed in', '/auth/google_oauth2' %>?</p>
```

Comment:
I'd be interested to know what @graygilmore thinks. I thought it looked a little funny with the punctuation in the link, but that's just my opinion - I'm not set on one way in particular. 
**Screenshots:**
https://drive.google.com/open?id=0B69-mH0H4uz4S0wtV2Z0T1pLd2M  vs https://drive.google.com/open?id=0B69-mH0H4uz4RE9IS0VvSUR0VkE